src/main/java/com/unstoppabledomains/util/DnsUtils.java

```diff
@@ -0,0 +1,117 @@
+package com.unstoppabledomains.util;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.google.gson.JsonSyntaxException;
+import com.unstoppabledomains.exceptions.DnsException;
+import com.unstoppabledomains.exceptions.DnsExceptionCode;
+import com.unstoppabledomains.exceptions.NSExceptionParams;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class DnsUtils {
+  
+  static Number DEFAULT_TTL = 300;
+  public List<DnsRecord> toList(Map<String, String> rawRecords) throws DnsException {
+    List<DnsRecord> dnsRecords = new ArrayList<DnsRecord>();
+    List<DnsRecordsType> types = getAllDnsTypes(rawRecords);
+    for (DnsRecordsType type: types) {
+      List<DnsRecord> dnsType = constructDnsRecords(rawRecords, type);
+      dnsRecords.addAll(dnsType);
+    }
+    return dnsRecords;
+  }
+
+  public Map<String, String> toMap(List<DnsRecord> records) throws DnsException {
+    Map<String, String> map = new HashMap<String, String>();
+    for (DnsRecord record: records) {
+      DnsRecordsType type = record.getType();
+      String ttlInRecord = map.get("dns." + type + ".ttl");
+      JsonArray dnsInRecord = getJsonArray(map.get("dns." + type), type);
+      if (dnsInRecord != null) {
+        dnsInRecord.add(record.getData());
+        map.put("dns." + type, dnsInRecord.toString());
```

Comment:
Please, make sure that the format of `toString()` gives you the output you're expecting.